stackable-base/Dockerfile

```diff
@@ -29,7 +29,7 @@ RUN cd ./config-utils && \
     cargo auditable build --release && cargo cyclonedx --output-pattern package --all --output-cdx
 
 # Manifest list digest because of multi architecture builds ( https://www.redhat.com/architect/pull-container-image#:~:text=A%20manifest%20list%20exists%20to,system%20on%20a%20specific%20architecture )
-FROM registry.access.redhat.com/ubi9/ubi-minimal@sha256:a7d837b00520a32502ada85ae339e33510cdfdbc8d2ddf460cc838e12ec5fa5a AS final
+FROM registry.access.redhat.com/ubi9-minimal@sha256:a7d837b00520a32502ada85ae339e33510cdfdbc8d2ddf460cc838e12ec5fa5a AS final
```

Comment:
Why these changes?

Reply:
They where not needed, we just wanted to switch to what we *think* is the new naming scheme to be safe going forward

Reply:
Then I changed it back in https://github.com/stackabletech/docker-images/pull/807 (because that's how it appears at the linked site I have added in my PR :shrug:)

Note: @sbernauer did ask me about this and I double checked the commit hashes match.